spec/classes/jmxtrans_spec.rb

```diff
@@ -61,6 +61,41 @@
           it { is_expected.to contain_service('jmxtrans') }
         end
 
+        context 'jmxtrans class with manage_service_file true' do
```

Comment:
There should also be the negative tests, that when `:manage_service_file => false`, it compiles but doesn't contain the relevant file resources.